src/rails_admin/styles/base/theming.scss

```diff
@@ -206,43 +126,21 @@ body.rails_admin {
     }
   }
 
-  // .modal {
-  //   width: 750px;
-  //   margin-left: -375px;
-  // }
-
-  /* nested nav */
-  .nav {
-    .nav-level-1 {
-      padding-left: 30px;
-    }
-    .nav-level-2 {
-      padding-left: 60px;
-    }
-    .nav-level-3 {
-      padding-left: 90px;
-    }
-  }
-
-  /* tab links should match icon color */
+  /* Action tabs */
   .nav.nav-tabs {
-    margin-bottom: 20px;
-
-    li.icon a {
-      color: #000;
+    li.nav-item.icon + li.nav-item.dropdown {
+      margin-left: auto;
     }
-  }
-
-  #sidescroll {
-    margin-bottom: 20px;
-    overflow-x: auto;
-    .table {
-      margin-bottom: 0;
+    /* To silence Popper's warning: 'CSS "margin" styles cannot be used...' */
```

Comment:
To fix the icon rendering against the `span` text in the nav
![Screen Shot 2022-01-07 at 13 14 18](https://user-images.githubusercontent.com/4504083/148608339-6c5de981-5d49-443b-8cc7-afcbcbb9149b.png)


```suggestion
    a > i {
      padding: 0 0.3rem;
    }
    /* To silence Popper's warning: 'CSS "margin" styles cannot be used...' */
```

though it may be worth considering having a high-level rule for `i` in general (to fix the same issue elsewhere in the UI, such as removing a filter option)
![Screen Shot 2022-01-07 at 13 14 22](https://user-images.githubusercontent.com/4504083/148608324-05a32151-1cfb-4931-ae75-542d87b56dd8.png)

Reply:
Adding a whitespace between icon and text might be the better way, since we're doing so a lot elsewhere.

Reply:
Sounds good (assuming that wouldn't cause issues with RTL-rendered content - which doesn't seem to be the case but mentioning it as a concern to be sure)